SDK/simVis/ProjectorManager.cpp

```diff
@@ -139,7 +139,16 @@ void ProjectorManager::registerProjector(ProjectorNode* proj)
 
     projectors_.push_back(proj);
 
+#if SDK_OSGEARTH_MIN_VERSION_REQUIRED(1,6,0)
+    ProjectorLayer* layer = new ProjectorLayer(proj->getId());
+    layer->setName("SIMSDK Projector");
+    osg::StateSet* projStateSet = layer->getOrCreateStateSet();
+    projectorLayers_.push_back(layer);
+
+    mapNode_->getMap()->addLayer(layer);
+#else
     osg::StateSet* projStateSet = new osg::StateSet();
+#endif
```

Comment:
Missing the previously existing layer->setStateSet(projStateSet)

In the new code, projStateSet leaks.

Reply:
That is, leaks if < 1.6

Reply:
my bad - fix coming

Reply:
Wait. Didn't it leak before?

Reply:
Yes, looks like it leaked before 1.6.  I'm OK with proceeding with this, or with a memory fix by wrapping in ref_ptr.  If you don't fix in your pull request no problem, we'll address in a future pass.  I'm OK either way.

Reply:
I'm happy to fix but not clear what the 1,6,0 conditional implies. My gut says you probably need the whole useRexEngine block in side that define.

Reply:
Because osgEarth is not versioned as often as we release, we attempt to determine the version relative to our releases.  So osgEarth-SDK 1.6 corresponds to the osgEarth API when we released SIMDIS SDK 1.6 (~March 2016).
The \#else block runs when that isn't met -- i.e. versions earlier than March 2016.
Our primary concern on support is matching the current osgEarth version (master), the version at latest release (1.7), and the version one back (1.6).  Your \#if matches all three cases.

If the new code should only be against current osgEarth master, then you'll need to \#if the setStateSet() call vs getOrCreateStateSet().

Internally we're still building against https://github.com/gwaldron/osgearth/releases/tag/osgearth-simdis-sdk-1.7, not osgEarth master.  I'm the only one (that I know of) that regularly updates to master.